src/main/java/au/org/democracydevelopers/raireservice/persistence/repository/ContestRepository.java

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2024 Democracy Developers
+The Raire Service is designed to connect colorado-rla and its associated database to
+the raire assertion generation engine (https://github.com/DemocracyDevelopers/raire-java).
+
+This file is part of raire-service.
+raire-service is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+raire-service is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more details.
+You should have received a copy of the GNU Affero General Public License along with raire-service. If not, see <https://www.gnu.org/licenses/>.
+*/
+
+package au.org.democracydevelopers.raireservice.persistence.repository;
+
+import au.org.democracydevelopers.raireservice.persistence.entity.Contest;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+/*
+ * Database retrieval for contests, either by name or by (CountyID, contestID) pairs.
+ */
+@Repository
+public interface ContestRepository extends JpaRepository<Contest, Long> {
+
```

Comment:
Can we make all method descriptions grammatically correct (eg. with full stop etc). Rather than using "Select" terminology, I think it would be clearer to say "Find and return all contests with a given name from the corla database." (Apply comment to later methods too).